docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of CentOS:
 
-- [*CentOS 7 (64-bit)*](#installing-docker-centos-7)
-- [*CentOS 6.5 (64-bit)*](#installing-docker-centos-6.5) or later
+* CentOS 7.X 
+* CentOS 6.5 or higher 
+
+Installation on other binary compatible EL6/EL7 distributions such as Scientific
+Linux might succeed, but Docker does not test or support Docker on these
+distributions.
+
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using CentOS-managed packages, consult your
+CentOS documentation.
+
+## Prerequisites
+
+Docker requires a 64-bit installation regardless of your CentOS version. Also,
+your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does
+not. We make an exception for CentOS 6.5. To run Docker on
```

Comment:
I'm not getting a clear direction here folks.  I would suggest we leave the content as is.  It is pretty clear --- I disagree with you there Tibor -- *we make an exception for* -- is unambiguous as written. As a policy, yes, I agree it is confusing. 

There is a certain set of people who want Docker and are willing to make sure their environment is correct. Those who are willing to hedge their bets will do so regardless of what we write.  I would also like to mark a clear version 6.5 .